test/module/irohad/simulator/simulator_test.cpp

```diff
@@ -131,7 +131,9 @@ auto makeTx() {
 
 TEST_F(SimulatorTest, ValidWhenPreviousBlock) {
   // proposal with height 2 => height 1 block present => new block generated
-  std::vector<shared_model::proto::Transaction> txs = {makeTx(), makeTx()};
```

Comment:
Could you please also address the problem of temporary protocol transactions captured by reference in these cases? It seems that this problem may arise somewhere without being noticed.